styles/gauntlet.css

```diff
@@ -84,7 +106,6 @@ COMBAT LAYOUT STYLES
 }
 
 #field {
-	height: 250px;
 	width: 90%;
 }
 
```

Comment:
Is there any way we could make this height responsive? Its way tall on my screen and throws off all the formatting.